### ISSUE TITLE
Annotations on migrate don't generate wrappers

### ISSUE THREAD

**Issue by amingilani** [ctran/annotate_models#399]
Annotations on migrate don't create wrappers specified in `auto_annotate_models.rake`. You have to run `annotate --force` to get them again.

**Comment by CyborgMaster** [comment]
I have the same problem.  Anytime `rake db:migrate` modifies an annotation, it removes the wrappers.  It does take an `annotate --force` to get them back.

**Comment by betesh** [comment]
:+1:

**Comment by thisismydesign** [comment]
This issue is still present for me in `v2.7.5`

**Comment by ibrahima** [comment]
Yeah... I'm also on 2.7.5 and I don't get annotations added without using `--force`, if the annotations are otherwise up to date.
